Backend/Models/OrderEntity/Order.cs

```diff
@@ -44,6 +44,7 @@ public class Order : ISoftDeletable
         [Required]
         public ICollection<OrderProductBalance> OrderProductBalances { get; set; }
 
+        [Required]
         public int? WarehouseId { get; set; }
```

Comment:
If it is required, then int should not be nullable

Reply:
fixed